fml/logging.cc

```diff
@@ -63,6 +64,9 @@ LogMessage::LogMessage(LogSeverity severity,
 LogMessage::~LogMessage() {
   stream_ << std::endl;
 
+  const auto log_string = stream_.str();
+  const char* chars = log_string.c_str();
```

Comment:
```
../../flutter/fml/logging.cc:68:15: error: unused variable 'chars' [-Werror,-Wunused-variable]
```

Reply:
Dang it I thought I fixed that. Thanks for catching.